### ISSUE TITLE
Is the custom ui working with lovelace?

### ISSUE THREAD

**Issue by fernmac2** [andrey-git/home-assistant-custom-ui#176]
Hello,
I am using hass.io 0.100.2 with lovelace and custom ui 20190518, I am nos sure if I am doing something wrong but after installing, loading and customizing entities, nothing happens.
Is it still working with latest lovelace?

**Comment by rccoleman** [comment]
Yes, it works for me.  First, you need to call homeassistant.reload_core_config or restart home assistant to load the customizations. The value/state of the entities that you’re customizing also need to change to reflect the customization, so you either need to wait for that, or call homeassistant.update_entity.

**Comment by fernmac2** [comment]
Thanks for answering.
Maybe I am doing something wrong, I installed and loaded as instructions, I restarted HA, in my customize.yaml file, I wrote for testing:

cover.estores1_dinner_room:
      custom_ui_state_card: state-card-custom-ui
      state_card_mode: break-slider

but I watch my frontend with the entity in question and nothing happens. The log as no error also.

**Comment by duffel90** [comment]
Sinz HA Version 100.2 this implementation don't work for me. Home Assisrant did not start because of an error in line 27 in custom state card ui.

**Comment by jtitley** [comment]
I've not been working on HA for almost two years. I've upgraded to 0.93.2, just after Lovelace. Nothing about CustomUI works any more. Not sure what to do. The most important one is hide_in_default_view.

Any advice on what might have changed?

**Comment by Mariusthvdb** [comment]
See https://github.com/Mariusthvdb/custom-ui

Btw you should really read up on the differences between Ha States and Lovelace.

Most functionality of ‘old’ custom_ui was for States. As the one you asked about.

**Comment by jtitley** [comment]
Looks clean, I can't wait. I must suffer through a Python 3.7 upgrade first though, considering how many versions behind I am.
